Localide/Classes/Localide.swift

```diff
@@ -22,7 +22,21 @@ internal protocol UIApplicationProtocol {
 
 extension UIApplication: UIApplicationProtocol {
     func localideOpen(_ url: URL, options: [OpenExternalURLOptionsKey : Any], completionHandler completion: ((Bool) -> Void)?) {
-        open(url, options: options, completionHandler: completion)
+        if url.absoluteString.starts(with: LocalideUrlStringAppleMaps.prefixesNew.rawValue) {
```

Comment:
Please make sure you also app the url scheme to the app info.plist of the Localide Demo
Also make sure to add it to the App2 info.plist / project.yml

![Screenshot 2025-05-25 at 11 26 19](https://github.com/user-attachments/assets/1898ccee-c97f-403f-849e-af2659ad3b95)